### PR TITLE
Csi 4747/support io groups svc implamantation

### PR DESCRIPTION
**Pull request by matancarmeli7**
related PRs:

- #610
- #611 
- #612 
- #613 
- #614 
- #615

### CODE REVIEW

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by ELENAGER — DISMISSED** [review]
LGTM

File: controllers/array_action/array_mediator_svc.py [hunk 1]
```diff
@@ -1841,11 +1851,57 @@ def get_host_connectivity_type(self, host_name):
...
         return None
 
+    def _raise_io_group_error(self, host_name, io_group, error_message):
+        self._raise_error_when_host_not_exist_or_not_meet_the_rules(host_name, error_message)
+        self._raise_invalid_io_group(io_group, error_message)
+
+    def _addhostiogrp(self, host_name, io_group):
+        cli_kwargs = {'iogrp': io_group}
+        try:
+            self.client.svctask.addhostiogrp(object_id=host_name, **cli_kwargs)
```
Comment (ArbelNathan) [inline controllers/array_action/array_mediator_svc.py:1861]: isn't just like
```suggestion
            self.client.svctask.addhostiogrp(object_id=host_name, iogrp=io_group)
```

Comment (matancarmeli7) [reply controllers/array_action/array_mediator_svc.py:1861]: yes

**Pull request merged**